designs/virtualization/4.5/Scheduling/Firmware-config/Firmware-config.md

```diff
@@ -0,0 +1,25 @@
+---
+parent: virtualization
```

Comment:
@andybraren - Is this case sensitive? I've seen most of the CNV stuff have "Virtualization"